client/src/app/tasks/types/response.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface ResponseInterface {
+  _id: number;
```

Comment:
Is it really number?

Reply:
Not it is not number, it is my mistake :)